### ISSUE TITLE
[Question] Can someone please explain why there are scaled terms in the existing RL examples?

### ISSUE THREAD

**Issue by celestialdr4g0n** [isaac-sim/IsaacLab#1781]
### Question

Could you explain why the Direct environment in the Franka cabinet setup includes a scaled DOF position in its observation space, while the same position term remains unscaled in the Manager-based environment? Specifically, what are the design or performance considerations that led to this difference?
Thanks in advance.
```
# Manager based
        joint_pos = ObsTerm(func=mdp.joint_pos_rel)
        joint_vel = ObsTerm(func=mdp.joint_vel_rel)
        cabinet_joint_pos = ObsTerm(
            func=mdp.joint_pos_rel,
            params={"asset_cfg": SceneEntityCfg("cabinet", joint_names=["drawer_top_joint"])},
        )
        cabinet_joint_vel = ObsTerm(
            func=mdp.joint_vel_rel,
            params={"asset_cfg": SceneEntityCfg("cabinet", joint_names=["drawer_top_joint"])},
        )
        rel_ee_drawer_distance = ObsTerm(func=mdp.rel_ee_drawer_distance)

        actions = ObsTerm(func=mdp.last_action
```

```
# Direct environment
    def _get_observations(self) -> dict:
        dof_pos_scaled = (
            2.0
            * (self._robot.data.joint_pos - self.robot_dof_lower_limits)
            / (self.robot_dof_upper_limits - self.robot_dof_lower_limits)
            - 1.0
        )
        to_target = self.drawer_grasp_pos - self.robot_grasp_pos

        obs = torch.cat(
            (
                dof_pos_scaled,
                self._robot.data.joint_vel * self.cfg.dof_velocity_scale,
                to_target,
                self._cabinet.data.joint_pos[:, 3].unsqueeze(-1),
                self._cabinet.data.joint_vel[:, 3].unsqueeze(-1),
            ),
            dim=-1,
        )
        return {"policy": torch.clamp(obs, -5.0, 5.0)
```

**Comment by Afshin1997** [comment]
If I am not wrong, in manager-based environments the *empirical_normalization* parameter is set to **False**, whereas in direct environments it is set to **True**. Consequently, in direct environments it is better to scale the observation elements to a reasonable range (e.g., [-1, 1]). This can be achieved by multiplying each observation element by an appropriate scaling factor to ensure more uniform normalization. These scaling factors can be defined as:

$scale = \frac{1}{max(element)}$

**Comment by celestialdr4g0n** [comment]
Thank you for your response, and I have no idea about empirical_normalization parameter. I heard of Reward Normalization for noise reduction, but I did not heard of State/Observation Normalization.
